### DISCUSSION TITLE
PanGestureHandler `waitFor` within a ScrollView.

### DISCUSSION THREAD

**Discussion by GiridharKarnik** [software-mansion/react-native-gesture-handler#1402] [Q&A]
I have multiple items in a scrollView and each item/row is encapsulated by a PanGestureHandler. When I try to scroll, the pan gesture handler picks up the touches. I have referenced the parent scrollView's ref in each of the childrens' PanGestureHandler's `waitFor`, but this does not seem to help.

I am pretty sure others have faced this issue before, how did you guys manage to solve this.

**Comment by ftanzarella-eviivo** [comment]
I have the same issue. Did you ever manage to solve this?

**Comment by npearson72** [comment]
Same issue here. Trying to understand how to prevent swipe X gesture while scrolling and visa-versa.
